using_images/s2i_images/php.adoc

```diff
@@ -120,8 +120,13 @@ The following environment variable sets its equivalent property value in the opc
 |Variable name |Description |Default
 
 |`*OPCACHE_MEMORY_CONSUMPTION*`
-|The OPcache shared memory storage size.
+|The link:http://php.net/manual/en/book.opcache.php[OPcache] shared memory storage size.
```

Comment:
I think you should also link to the setting itself: https://secure.php.net/manual/en/opcache.configuration.php#ini.opcache.memory-consumption